types/inquirer/inquirer-tests.ts

```diff
@@ -1,4 +1,3 @@
-import "./test";
```

Comment:
I'm not sure if this is going to work the way you think it is. If you look [here](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package), you'll see that `foo-tests.ts` is the only file that will be tested; thus, the one that needs to do the importing.

Just thinking we might get some false positives with this new way you're doing it. Can you please just leave the tests as is? I understand your motivation to make these changes, but in fear that it goest against the DT guidelines, I think we should avoid this type of change.

Reply:
@jedmao no worries
Some type-declarations such as [athenajs](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/athenajs) use the same approach.

Do you rather prefer the splitted version of the tests or should I put them back into the Tests-file?

Reply:
I mean, it looks promising, but I don't see it documented anywhere. I'd feel a lot better if this PR were 2 commits, where the first didn't move the files around and the 2nd was JUST the files being moved around. I'd also like to see the green checkmark on the 1st commit, most importantly.

Reply:
Unfortunately, the logging on Travis-CI isn't verbose enough to show me that these exact files were actually tested; only, they at least show up in the diff.

Reply:
Aah I got your point
I just tested whether errors in one of the `types/inquirer/test/*.ts`-files are reported when running either `npm run lint inquirer` or `npm test`.

These files are considered in the test-commands as they are added to the `files`-property in the `types/inquirer/tsconfig.json`-file.

Also files placed in a directory called `test/` or files which are called `{package-name}-tests.ts(x)?` are considered as test-files:
https://github.com/microsoft/types-publisher/blob/c4762a372a40fa3dd26f7b7308d8257ba551e093/src/lib/definition-parser.ts#L237-L251